ios/Video/RCTVideo.m

```diff
@@ -512,16 +594,21 @@ - (void)setupNowPlaying {
     NSLog(@"RCTVideo setupNowPlaying, _playInBackground:%d", _playInBackground);
     MPNowPlayingInfoCenter *playingInfoCenter = [MPNowPlayingInfoCenter defaultCenter];
     
-    NSMutableDictionary *songInfo = [[NSMutableDictionary alloc] init];
+    //get existing song info dictionary or create new one
+    NSMutableDictionary *songInfo = (playingInfoCenter.nowPlayingInfo != nil) ? [[NSMutableDictionary alloc] initWithDictionary: playingInfoCenter.nowPlayingInfo] : [NSMutableDictionary dictionary];
 
     [songInfo setObject:_title forKey:MPMediaItemPropertyTitle];
     if(_artist) {
         [songInfo setObject:_artist forKey:MPMediaItemPropertyArtist];
     }
     
-    [songInfo setObject:[NSNumber numberWithDouble:_playerItem.currentTime.value] forKey:MPNowPlayingInfoPropertyElapsedPlaybackTime];
-    [songInfo setObject:[NSNumber numberWithDouble:_playerItem.duration.value] forKey:MPMediaItemPropertyPlaybackDuration];
-    [songInfo setObject:[NSNumber numberWithDouble:(_player.rate ? 0.0f : 1.0f)] forKey:MPNowPlayingInfoPropertyPlaybackRate];
+    double playerRate = (double) _player.rate;
+    double playerCurTime = (double) CMTimeGetSeconds(_playerItem.currentTime);
```

Comment:
using `CMTimeGetSeconds` was the key to resolving timeline playback issues in nowPlaying lock controls.